config/locales/en/faker/offence.yml

```diff
@@ -810,7 +810,7 @@ en:
         - Defendant chooses trial by court martial 1
         - Defendant elects trial by jury
         - Summary-only offence
-        - Youth - offence triable on indictment only (homicide / relevant firearms / dangerousness)
+        - Youth - offence triable on indictment only (homicide / relevant firearms dangerousness)
```

Comment:
This needs reverting 
`- Youth - offence triable on indictment only (homicide / relevant firearms / dangerousness)`